website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -582,7 +694,8 @@ The default installation doesn't have data compression configured for Nginx ingr
 2. Add the following to the data section.
 
   ```yaml
-  data: # ADD IF NOT PRESENT
+  # Add the data section if not already present
+  data: 
```

Comment:
Can we move the comment `# Add the data section...` outside the code block and into the above point text please? People often copy-paste contents of code-block, which means they'll also copy this comment. But this comment won't hold value, when pasted in the resource file. It's just documentation text.